examples/export_runs.py

```diff
@@ -0,0 +1,52 @@
+from domino import Domino
+import pandas as pd
+import json
+import os
+
+output_dir = "results"
+
+# connect to domino; be sure to have these environment variables set
+#  (runs inside a Domino executor automatically set these for you)
+domino = Domino("nick/digits", api_key=os.environ['DOMINO_USER_API_KEY'], host=os.environ['DOMINO_API_HOST'])
+raw_runs = domino.runs_list()['data']
+
+# print number of runs to STDOUT
+print "Details of {0} runs received".format(len(raw_runs))
+
+# create results directory if it doesn't exist
+if not os.path.exists(output_dir):
+  os.makedirs(output_dir)
+
+# write API out to all_runs.json
+f = open('{0}/all_runs.json'.format(output_dir), 'w')
+f.write(json.dumps(raw_runs))
```

Comment:
What's the function of outputting all the runs as JSON? Is this just for debugging or am I missing something here?